ci/jobs/scripts/clickhouse_proc.py

```diff
@@ -562,7 +562,16 @@ def create_minio_log_tables(self):
             )
         )
         res = "success" in status
-        print(f"Clickminio restart status: {status}, res: {res}")
+        if res:
+            for _ in range(20):
+                res = Shell.check(
+                    "/mc ls clickminio/test | grep -q .",
```

Comment:
```suggestion
                    "/mc ls clickminio/test",
```

Why do we need this grep at all?

Reply:
to keep logs clean

Reply:
Thanks missed the `-q`, but sometimes more logs we have the better